### DISCUSSION TITLE
My Long-Distance University is not considered as such!

### DISCUSSION THREAD

**Discussion by MimisTh** [community/community#143049] [GitHub Education]
### Hi everyone,

Hi. After trying to apply for the github student developer pack, my request gets rejected constantly, due to my location not being that of the university(Hellenic Open University) I am studying in. The issue probably comes from the fact that the university also has a physical campus, one which is not near my location. Therefore, my request gets rejected, even though I provided paperwork which states the remote nature of the university. I'd like your help in resolving this issue.

(below is official paperwork stating I am studying at the uni remotely)
[ΠΛΗ_std168214@ac.eap.gr_signed.pdf](https://github.com/user-attachments/files/17567276/_std168214%40ac.eap.gr_signed.pdf)

**Comment by davevad93** [comment]
Hi @MimisTh ,

Try contacting support by opening a ticket [here](https://support.github.com/contact/education), select "`My selected school has incorrect or incomplete information e.g. domains or campus location`" and explain your issue.
